test(allocator): ignore a slow doc test

Dunqing opened the pull request:
This doc test runs a few times `workload`, which is quite a heavy function. Ignore this test, we can save ~1.5s
https://github.com/oxc-project/oxc/blob/3503d708ad7221109bde34bea129149f7170d42c/crates/oxc_allocator/src/allocator.rs#L112-L117

Dunqing commented:
* **#10521** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10521?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/10521?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=10521).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented:
@overlookmotel I think we need to ask you for permission on this one 😅

overlookmotel reviewed (approved):
Ah yes, my mistake, I didn't clock how slow this would be. I wrote it as docs, not a doc test. Ignore it!

overlookmotel commented:
### Merge activity

* **Apr 22, 4:48 AM EDT**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Dunqing committed:
test(allocator): ignore a slow doc test (#10521)

This doc test runs a few times `workload`, which is quite a heavy function. Ignore this test, we can save ~1.5s
https://github.com/oxc-project/oxc/blob/3503d708ad7221109bde34bea129149f7170d42c/crates/oxc_allocator/src/allocator.rs#L112-L117
